Makefile

```diff
@@ -0,0 +1,7 @@
+GOCMD=go
+GOTEST=$(GOCMD) test
+
+# the standard vet set is atomic,bool,buildtags,nilfunc,printf. printf causes a lot of spurious failures, so leave that
+# out
+test:
+	$(GOTEST) -vet=atomic,bool,buildtags,nilfunc ./...
```

Comment:
This is a good idea, thanks. Can you mention it in the README?

Reply:
Sure thing!